test/Conversion/SCFToCalyx/convert_memory.mlir

```diff
@@ -460,14 +460,14 @@ module {
 
 // CHECK:         calyx.group @bb0_0  {
 // CHECK-NEXT:         calyx.assign %std_slice_1.in = %in0 : i32
-// CHECK-NEXT:         calyx.assign %mem_0.addr0 = %std_slice_1.out : i0
+// CHECK-NEXT:         calyx.assign %mem_0.addr0 = %std_slice_1.out : i1
```

Comment:
I think this is sufficient. This test below is not really necessary. If you want to keep it, please reduce the number of CHECKs, since all we care about is that the dimensions is 0 and not 1.

Reply:
Actually I rescind that. We should keep your test, in case this one is removed for whatever reason.

Can you just add something like:

```mlir
// CHECK-DAG:       %std_slice_3.in, %std_slice_3.out = calyx.std_slice @std_slice_3 : i32, i1
// CHECK-DAG:       %std_slice_2.in, %std_slice_2.out = calyx.std_slice @std_slice_2 : i32, i1
```

to a few wires that would have been zero without these changes?